### DISCUSSION TITLE
About Formik v3, is it released? any plan?

### DISCUSSION THREAD

**Discussion by zlf341** [jaredpalmer/formik#3220] [General]
Hi, guys

I am appreciated what you guys do for formik, which is a great lib, 

I am just wondering when do we have Formik v3? 
looks no update since Dec 2020. 

any plans for it?

thank you

**Comment by dantman** [comment]
The best thing to watch to keep up to date seems to be #3099 and #3231.

The current status seems to be that there is a working non-production alpha with mainly minor fixes and docs/administrative stuff left. But no momentum on moving it forward.

**Reply by johnrom** [reply]
I have pinged multiple times and heard very little in terms of responses. For now, I'm just using my fork 🤷 . My recommended course of action is either:

a. a release candidate release where users can try an official build via npm
b. release and let RN users know that we haven't been able to fully vet the changes there

Either way the documentation needs to be written but since the major components aren't changing, I think we can just write a small release post about it letting people know they'll need to useFormikState() instead of using `formik.values`, etc, directly from useFormikContext(), and follow up with a major documentation sprint.

(assuming the project ever gets moving again)

**Reply by pamapa** [reply]
Yes please release a release candidate, such that people can give you feedback...
